### ISSUE TITLE
ApplicationGuard compatibility

### ISSUE THREAD

**Issue by joestern** [MicrosoftDocs/windows-itpro-docs#5947]
I initially had trouble signing up for the 60-day trial because I needed to add marketing.plutoserv.com to the list of ApplicationGuard-compatible domains. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d6d1d825-ab12-2f37-b692-5266487ef16f
* Version Independent ID: fef750ac-311b-28b4-1661-16385997815b
* Content: [Web content filtering - Windows security](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/web-content-filtering#feedback)
* Content Source: [windows/security/threat-protection/microsoft-defender-atp/web-content-filtering.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/microsoft-defender-atp/web-content-filtering.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @levinec
* Microsoft Alias: **ellevin**

**Comment by VLG17** [comment]
@joestern Could you give more details around "I initially had trouble signing up for the 60-day trial". Were you getting an error? If yes, what error? Were you seeing something else?

Also, where exactly did you add marketing.plutoserv.com and where did you get this information from, that you need to add it? I cannot find anything around this topic.

**Comment by joestern** [comment]
My org uses [Windows Defender Application Guard](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-guard/configure-wd-app-guard) on our privileged administrator workstations, like the one I use to perform administration of ATP services. With WDAG, any Internet domain that is not explicitly allowed is opened inside a virtual machine sandbox and cannot exchange information with the host. When I tried to sign up for the trial, I was starting from an approved domain, probably something in Azure or Microsoft that I had already whitelisted. The signup process redirected me to marketing.plutoserv.com, and that page opened up in an orphaned state inside a WDAG browser. In order to get signed up for this trial, I had to add that domain to my list of approved domains in Computer Configuration\Administrative Templates\Network\Network Isolation. 

Somehow it took me longer than usual to figure out why the sign-up was failing, which is why I added the note to the github page.
